README.md

```diff
@@ -103,4 +103,8 @@ All other files are only available under CC BY 4.0, including:
 + `*.pdf`
 + `*.docx`
 
+Third-party code under an MIT license:
```

Comment:
Perhaps change this to

```md
Except for the following files with different licenses:

+ `build/assets/anchors.js` which is released under an [MIT License](https://opensource.org/licenses/MIT).
```

This way we can add additional files with their own distinct licenses. Does that make sense?

Reply:
I like @dhimmel's extended version.  We could also link back to the original source in the readme even if the MIT license requirements are already satisfied in `anchors.js`.  For example, change `build/assets/anchors.js` to [`build/assets/anchors.js`](https://github.com/bryanbraun/anchorjs).

Reply:
I don't like linking what seems to be a file path to a GitHub repo landing page. If you want to attribute the source here, which I don't is necessary but I'm not against, you could hyperlink the word released rather than the file path. We can link the file path, but it should be a relative link to `build/assets/anchors.js`, which will bring someone to the file.

Reply:
Added with your suggestions, thanks! I've linked to the Anchorjs landing page though.